sdk/core/System.ClientModel/README.md

```diff
@@ -12,7 +12,7 @@ it will be installed for you when you install one of the client libraries using
 Install the client library for .NET with [NuGet](https://www.nuget.org/packages/System.ClientModel).
 
 ```dotnetcli
-dotnet add package System.ClientModel --prerelease
+dotnet add package System.ClientModel
```

Comment:
If it's not too much work, it might be worth adding a fix for this issue https://github.com/Azure/azure-sdk-for-net/issues/40306 to this PR.

Reply:
Added the two that exist